manifests/mpm.pp

```diff
@@ -101,11 +101,21 @@
 
       if $mpm == 'prefork' {
```

Comment:
```suggestion
      if $mpm in ['prefork', 'worker'] {
```

Reply:
AFAIK: prefork must disable event in Debian 9, whereas worker must disable event in Debian 8.
But I might be wrong. We had this issue at a customer today.

Reply:
@ekohl can you please review?